### PR TITLE
New vmmlib API

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: brain/neuron/soma.h [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 namespace brain
 {
+/** High-level interface to neuron morphologies. */
```
Comment (tribal-tec) [inline brain/neuron/soma.h]: moved that to types.h? namespace brain brief is there as well

**Comment by tribal-tec** [comment]
+1 otherwise

File: brain/neuron/soma.h (hunk 1 above)
Comment (eile) [inline brain/neuron/soma.h]: why is there no brain/neuron/types.h.... sigh.

Comment (eile) [inline brain/neuron/soma.h]: done

**Pull request merged**